Sources/Administrator/InputHandler.cpp

```diff
@@ -0,0 +1,121 @@
+//
+// Created by daria on 21.05.16.
+//
+
+#include "InputHandler.h"
+#include <iostream>
+#include <cstring>
+
+void InputHandler::run()
+{
+    std::cout << "Waiting for a command:\n";
+    std::string command;
+    std::string name;
+
+    while ( name != "exit")
+    {
+        std::getline (std::cin,command);
+
+        if ( command.find(" ") == std::string::npos )
+            name = command;
+        else name = command.substr(0,command.find_first_of(" "));
+
+        if ( name == "help") {
+        printHelp();
+        }
+        else if ( name == "connect") {
+        handleConnect(command);
+        }
+        else if ( name == "disconnect") {
+        handleDisconnect();
+        }
+        else if ( name == "send_process" ) {
+        sendProcess(command);
+        }
+        else if (  name == "launch_process" ) {
+        launchProcess(command);
+        }
+        else {
+        std::cout << "Incorrect command.\n";
+        printHelp();
+        }
+    }
+    std::cout << "Exiting Administrator process. Goodbye.\n";
+}
+
+
+void InputHandler::printHelp()
+{
```

Comment:
I checked other files and there wasn't namespace used so I thought that we shouldn't use it :/